From kay.sievers@vrfy.org  Thu Apr 16 11:01:04 2009
From: Kay Sievers <kay.sievers@vrfy.org>
Date: Thu, 16 Apr 2009 19:56:37 +0200
Subject: driver core: set default SYSFS_DEPRECATED=n
To: gregkh <gregkh@suse.de>
Message-ID: <1239904597.17211.20.camel@poy>


From: Kay Sievers <kay.sievers@vrfy.org>

All recent distros depend on the non-deprecated sysfs layout, so
change the default value of the option to reflect that.

Signed-off-by: Kay Sievers <kay.sievers@vrfy.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 init/Kconfig |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/init/Kconfig
+++ b/init/Kconfig
@@ -616,13 +616,13 @@ config SYSFS_DEPRECATED
 	bool
 
 config SYSFS_DEPRECATED_V2
-	bool "Create deprecated sysfs layout for older userspace tools"
+	bool "remove sysfs features which may confuse old userspace tools"
 	depends on SYSFS
-	default y
+	default n
 	select SYSFS_DEPRECATED
 	help
 	  This option switches the layout of sysfs to the deprecated
-	  version.
+	  version. Do not use it on recent distributions.
 
 	  The current sysfs layout features a unified device tree at
 	  /sys/devices/, which is able to express a hierarchy between
